markpbaggett/exodus
# Change "attribute" from "edtf" to "encoding."

## What does this Pull Request do?

This PR corrects an Xpath error for date_created, date_created_d, date_issued, and date_issued_d. The value "edtf" was incorrectly placed as the attribute name instead of "encoding."

It also updates [03_adding_collections.rst](https://raw.githubusercontent.com/markpbaggett/exodus/main/docs/source/contents/03_adding_collections.rst) so that it reads from [get_collection_data.py](https://github.com/markpbaggett/exodus/blob/main/exodus/helpers/get_collection_data.py) rather than relying on hard coding.

## How should this be tested?

Check to make sure the Xpaths are correct. Rerun a spreadsheet and see if edtf dates in metadata go to the machine readable dates properties.

## Additional Notes:

I'll be opening a second PR focused on creators and contributors. I think there's more likely to be issues and discussion there, so I feel like it's best to separate these commits into separate PRs.

## Interested parties

@markpbaggett
